htlcswitch/link.go

```diff
@@ -843,35 +843,48 @@ func (l *channelLink) fwdPkgGarbager() {
 	l.cfg.FwdPkgGCTicker.Resume()
 	defer l.cfg.FwdPkgGCTicker.Stop()
 
+	if err := l.loadAndRemove(); err != nil {
+		l.log.Warnf("unable to load and remove initial fwd pkgs for "+
```

Comment:
nit: how about "unable to run initial fwd pkgs gc: %v"

Reply:
Yup